bin/geni-sync-wireless

```diff
@@ -0,0 +1,618 @@
+                ldif_text = \
+                    self._orb.ldif_for_user(orbit_username,
+                                            self._options.holdingpen_group,
+                                            member_pretty_name,
+                                            first_name,
+                                            member_info['email_address'],
+                                            last_name,
+                                            member_ssh_keys,
+                                            self.holdingpen_group_description,
+                                            None)
+                self._orb.saveUser(ldif_text)
+
+    
+    # Make sure all wimax-enabled GENI projects have a corresponding 
+    # ORBIT group
+    def ensure_projects_exist(self): 
+        for project_id, project_info in self._geni_projects.items():
+            project_name = project_info['project_name']
+            project_description = project_info['project_description']
+            orbit_group_name = self.to_orbit_name(project_name)
+            if orbit_group_name not in self._orbit_groups:
+                print "Creating ORBIT group: %s" % orbit_group_name
+                lead_id = project_info['lead_id']
+                lead_username = self._geni_members[lead_id]['username']
+                orbit_lead_username = self.to_orbit_name(lead_username)
+                ldif_text = self._orb.ldif_for_group(orbit_group_name, 
+                                               project_description)
+                ldif_text = ldif_text + \
+                    self._orb.ldif_for_group_admin(orbit_group_name, 
+                                             orbit_lead_username,
+                                             self._options.holdingpen_group)
+                self._orb.saveUser(ldif_text)
+
+                # Add new group to self._orbit_groups structure
+                # Leave users blank so we'll re-create them later
+                orbit_group_info = {'admin' : orbit_lead_username,
+                                    'users' : []}
+                self._orbit_groups[orbit_group_name] = orbit_group_info
+
+    # Make sure all members of wimax-enabledf GENI projects are membes
+    # of the corresponding ORBIT group
+    # Enable all users that are members of a non-holdingpen group
+    def ensure_project_members_in_groups(self): 
+        users_to_enable = set()
+        for project_id, project_info in self._geni_projects.items():
+            project_name = project_info['project_name']
+            orbit_group_name = self.to_orbit_name(project_name)
+            group_info = self._orbit_groups[orbit_group_name]
+            for member_id in project_info['members']:
+                if member_id not in self._geni_members: continue
+                member_info = self._geni_members[member_id]
+                member_ssh_keys = member_info['ssh_keys']
+                if len(member_ssh_keys) == 0: continue
```

Comment:
We're logging it in ensure_project_members_exist, which is always going to be called in tandem with this, so we'll get that log message.